Cargo.toml

```diff
@@ -18,6 +18,7 @@ rust-version = "1.65"
 
 [dependencies]
 subtle = { version = "2.4", default-features = false }
+paste = "1.0.12"
```

Comment:
I would prefer not to add any additional mandatory dependencies.

Also, this use case doesn't seem to warrant an optional dependency either.